README.MD

```diff
@@ -139,3 +139,13 @@ The built artifacts for this module are available.
 See [configuration](https://dev.folio.org/download/artifacts) for repository access,
 and the [Docker image](https://hub.docker.com/r/folioorg/mod-circulation-storage/).
 
+## Design notes
+
+### Request status transition
```

Comment:
Please can some context be added to this, to refer to the report in mod-circulation and the reason why this has been implemented here, rather than in mod-circulation